blog/2024-04-30-introducing-opencost-plugins/index.mdx

```diff
@@ -0,0 +1,155 @@
+        "charge_category": "",
+        "description": "",
+        "resource_name": "",
+        "resource_type": "",
+        "provider_id": "",
+        "billedCost": 147.37999,
+        "listCost": 186.98547,
+        "list_unit_price": 0.082197905,
+        "usage_quantity": 120,
+        "usage_unit": "",
+        "domain": "datadog",
+        "cost_source": "observability",
+        "aggregate": "datadog"
+      }
+    ]
+  }
+}
+```
+
+
+_Totals request, aggregating by providerId over the past 7 days:_
+
+
+```
+https://nightly.opencost.io/customCost/total?window=7d&aggregate=providerId
+```
+
+Response:
+
+```
+{
+  "code": 200,
+  "data": {
+    "window": {
+      "start": "2024-03-14T00:00:00Z",
+      "end": "2024-03-21T00:00:00Z"
+    },
+    "totalBilledCost": 147.37999,
+    "totalListCost": 186.98546,
+    "customCosts": [
+      {
+        "id": "",
+        "zone": "us",
+        "account_name": "Kubecost",
+        "charge_category": "usage",
+        "description": "350+ integrations, alerting, custom metrics & unlimited user accounts",
+        "resource_name": "agent_host_count",
+        "resource_type": "infra_hosts",
+        "provider_id": "42c0ac62-8d80-11ed-96f3-da7ad0900005/agent_host_count",
+        "billedCost": 0,
+        "listCost": 8.876712,
+        "list_unit_price": 0.073972605,
+        "usage_quantity": 360,
+        "usage_unit": "Infra Host - hours",
+        "domain": "datadog",
+        "cost_source": "observability",
+        "aggregate": "42c0ac62-8d80-11ed-96f3-da7ad0900005/agent_host_count"
+      },
+      {
+        "id": "",
+        "zone": "us",
+        "account_name": "Kubecost",
+        "charge_category": "usage",
+        "description": "Centralize your monitoring of systems and services (Per Container)",
+        "resource_name": "container_count_excl_agent",
+        "resource_type": "infra_hosts",
+        "provider_id": "42c0ac62-8d80-11ed-96f3-da7ad0900005/container_count_excl_agent",
+        "billedCost": 0,
+        "listCost": 19.80274,
+        "list_unit_price": 0.0041095894,
+        "usage_quantity": 14456,
+        "usage_unit": "Container - hours",
+        "domain": "datadog",
+        "cost_source": "observability",
+        "aggregate": "42c0ac62-8d80-11ed-96f3-da7ad0900005/container_count_excl_agent"
+      }
+...
+    ]
+  }
+}
+```
+
+Datadog is just the first of many plugins! If you are interested in a plugin to monitor your resources, you can review the process for [adding a new plugin](https://github.com/opencost/opencost-plugins?tab=readme-ov-file#creating-a-new-plugin) and make a contribution, or [open an issue](https://github.com/opencost/opencost-plugins/issues) requesting a new plugin. The first 10 contributors who add a plugin which is accepted into Opencost will receive swag and $1,000! If you have any issues or questions you can [join our slack community](https://slack.cncf.io/) or email us at [opencost@kubecost.com](mailto:opencost@kubecost.com).  

```

Comment:
@kwombach12 @bstuder99 should we potentially point them to open issues here for requests here? https://github.com/opencost/opencost/issues

i know it's not necessarily the right solution, but SO many more people are likely to see these and this will help with opencost-plugin discovery...